lib/src/widgets/reactive_form_array.dart

```diff
@@ -57,8 +57,8 @@ class _ReactiveFormArrayState<T> extends State<ReactiveFormArray<T>> {
       _formArray = widget.formArray!;
     } else {
       final form =
-          ReactiveForm.of(context, listen: false) as FormControlCollection;
-      _formArray = form.control(widget.formArrayName!) as FormArray<T>;
+          ReactiveForm.of(context, listen: false) as FormControlCollection?;
+      _formArray = form!.control(widget.formArrayName!) as FormArray<T>;
```

Comment:
I don't understand this change, is this just a matter of preferences, or it means a real change?